src/afl-performance.c

```diff
@@ -425,3 +425,15 @@ char *sha1_hex_for_file(const char *fname, u32 len) {
 
 }
 
+#ifdef _DEBUG
+u32 hash32_xxh32(u8 *key, u32 len, u32 seed) {
```

Comment:
the reason we use xxh64 and just use 32 bit is that I read that it is faster than xxh32. if you tested that xxh32 is faster for you?

Reply:
During our test, different hashing algorithms (we tested xxhash series and murmurhash and a few other candidates) almost has no performance impact, at least less than the random error across repetitions. We only cares about the collision rate and the final implementation here works the best during our evaluation.

Reply:
have you already tested this?

Reply:
Not yet, will update once done.

Reply:
I decided to use AFL++ `hash32` due to lacking time to evaluate this. I recall that there is no significant performance issue.